0-bootstrap/modules/jenkins-agent/README.md

```diff
@@ -188,69 +191,104 @@ You will need this key pair to enable authentication between the Master and Agen
     -----END RSA PRIVATE KEY-----
     ```
 
-**1.2 Configure a new SSH Jenkins Agent.**
+#### 1.2. Configure a new SSH Jenkins Agent
 
 In the Jenkins Master’s Web UI, use the following information to configure a new [SSH Jenkins Agent](https://plugins.jenkins.io/ssh-agent/):
 - SSH private key you just generated
 - The passphrase that protects the private key if you used the `-N ""` option
 - The Jenkins Agent’s private IP address assigned by your Network Administrator
 
-**1.3 Configure the Git repository in your Jenkins Master.**
-
-In your Jenkins Master, you might need to configure credentials to connect to the Git repository where you plan to store your infrastructure code.
-- You may also want to configure an automatic build trigger. Note that this is considered out of the scope of this document, since there are multiple options on how to do this and multiple Jenkins Plugins able to help with the task. You can also simply run the build job manually from the Jenkins Web UI.
-
-### **2 Bootstrap scripted steps using terraform**
+#### 1.3. Clone this repository in multiple repositories
 
 - Required information:
-  - Four Git repositories: one repo for each directory in this [monorepo](https://github.com/terraform-google-modules/terraform-example-foundation) (`0-bootstrap, 1-org, 2-environments, 3-networks, 4-projects`)
-  - Public SSH key generated in the Jenkins Master (this is not a secret and can be in your repository code). Copy the public key using: `cat "${SSH_KEY_FILE_PATH}.pub"`
-  - VPN details from your Network administrator:
-     - VPN public IP
-     - VPN PSK secret
-     - Private IP address for the Jenkins Agent (which will be created in the `cicd` GCP Project in Step "2.4. Run terraform commands"). This private IP will be reachable through the VPN connection that you will create in Step "3.2. Configure VPN Network tunnel".
+  - Public SSH key generated in the Jenkins Master (this is not a secret and can be in your repository code). Show the public key using `cat "${SSH_KEY_FILE_PATH}.pub"`, you will have to copy / paste it in the the `terraform.tfvars` file.
 
-**2.1. Clone the Repository provided here**
+Although this infrastructure code is distributed to you as a [monorepo](https://github.com/terraform-google-modules/terraform-example-foundation), you will store it in five different repositories, one for each directory (`0-bootstrap, 1-org, 2-environments, 3-networks, 4-projects`). Here we will work with the first of these repositories <YOUR_NEW_REPO_CLONE-0-bootstrap>.
 
+- 1.3.1. Clone this repository:
```

Comment:
We usually try to avoid numbering that is not incremented automatically.

Reply:
Done. Thank you.